librz/main/rz-find.c

```diff
@@ -140,6 +142,15 @@ static int hit(RzSearchKeyword *kw, void *user, ut64 addr) {
 			}
 		}
 	}
+	if (ro->exec_command) {
+		char *command = rz_str_newf("%s %s", ro->exec_command, ro->curfile);
```

Comment:
Filename should be put in quotes here to avoid (likely unintended) command injection and properly handle filenames with spaces.

Reply:
Okay Sure

Reply:
> Filename should be put in quotes here to avoid (likely unintended) command injection and properly handle filenames with spaces.

actually the only way to patch this is to use the `rz_sys_execve` but i would prefer to merge this as is